### PR TITLE
Ecoinvent energy

### PR DESCRIPTION
**Pull request by ccomb**
Preleminary work on Textile export with Ecoinvent energy processes computed by brightway. This is not yet enabled because of the current weird impacts.

[User story](https://www.notion.so/fd0068a7db8348c584cd53a3eea399f2?v=37bf8aeb782747c3bf744ecaeafefd04&p=b683e3126d814191b340397666ea2d5a&pm=s)
New version of the brightway activity explorer with tabs and buttons in the technosphere

### CODE REVIEW

File: data/import_method.py [hunk 1]
```diff
@@ -1,16 +1,12 @@
...
 
-# import functools
-
-PROJECT = "Food"
+PROJECT = sys.argv[1]
 # Agribalyse
-DATAPATH = "AGB3.1.1.20230306.CSV.zip"
-DBNAME = "Agribalyse 3.1.1"
-BIOSPHERE = DBNAME + " biosphere"
+BIOSPHERE = "Agribalyse 3.1.1 biosphere" if PROJECT == "Food" else "biosphere3"
```
Comment (magopian) [inline data/import_method.py:9]: Should there be some kind of warning or help message if there's a typo instead of defaulting to textile?

**Review by magopian — APPROVED** [review]
That's a huge work done, congratz!
I haven't tested the exports/imports myself, so can't comment on that.

File: public/data/textile/materials.json [hunk 2]
```diff
@@ -79,22 +79,6 @@
...
-    "spinningProcessUuid": "4c5438c3-c360-413c-b357-fba2a851e44c",
-    "name": "Fil de jute / kenaf",
-    "shortName": "Jute / kenaf",
-    "origin": "Naturelles",
-    "primary": false,
-    "geographicOrigin": "Asie - Pacifique",
-    "defaultCountry": "CN",
-    "priority": 0,
-    "cff": null
-  },
```
[the remark this answers is not in the document]
Comment (ccomb) [reply public/data/textile/materials.json:97]: thx, I must have skipped recent addition

File: data/import_method.py (hunk 1 above)
Comment (ccomb) [reply data/import_method.py:9]: I'll remove that because I'm testing moving everything to biosphere3

**Comment by paulboosz** [comment]
One thing left, before we could select the energy source between the following process aliases : 
- steam-coal-rsa
- steam-coal-rer
- steam-gas-rer
- steam-gas-rsa
- steam-light-fuel-rer
- steam-light-fuel-rsa
- steam-heavy-fuel-rer
- steam-heavy-fuel-rsa

Using now the ecoinvent processes we only have : 
- steam-coal-rsa
- steam-coal-rer
- steam-gas-rer
- steam-gas-rsa

So we're reducing the energy choice between gas and coal. That has been validated but we need to make sure it doesn't break anything

**Pull request merged**